### PR TITLE
Virt pools

### PR DESCRIPTION
**Pull request by cbosdo**
Documentation for the new Virtual storage pool UI and uyuni-project/uyuni#1166

### CODE REVIEW

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 1]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+=== Refreshing a pool
+
+[systemitem]``Libvirt`` doesn't update the pool usage and contents in real time.
+In order to update the usage or to see a volume that has been created outside of {productname}, the pool needs to be refreshed.
+Click the [guimenu]``Refresh`` button to schedule a refresh of the pool.
+
+=== Creating a pool
+
+To create a storage pool click the [guimenu]``Create Pool`` button.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]: This should be a procedure.

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 2]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+[guimenu]``Add to``::
+Add the pool creation action to a new or already created action chain.
+
+All the remaining fields are depending on the selected field type.
+These fields are grouped in two panels:
+
+* the [guimenu]``Source`` panel contains the data about the device holding the pool.
+* the [guimenu]``Target`` panel contains the data about where to find the pool on the virtual host.
+
+==== Source fields
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
==== Source Fields
```

I'm not sure this needs to be a heading.

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 3]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+[guimenu]``Device path``::
+Path to a device containing the pool data
+
+[guimenu]``Partition separator``::
+Tell libvirt to use ``'p'`` as a partition separator in the path name.
+
+[guimenu]``Format``::
+Select the format of the pool source.
+The values from this field are depending on the pool type.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
The available values depend on the pool type.
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 4]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+* the [guimenu]``Source`` panel contains the data about the device holding the pool.
+* the [guimenu]``Target`` panel contains the data about where to find the pool on the virtual host.
+
+==== Source fields
+
+[guimenu]``Device path``::
+Path to a device containing the pool data
+
+[guimenu]``Partition separator``::
+Tell libvirt to use ``'p'`` as a partition separator in the path name.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
Use ``'p'`` as a partition separator in the path name.
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 5]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+The values from this field are depending on the pool type.
+
+[guimenu]``Host name``::
+IP or FQDN of the remote machine providing access to the pool.
+
+[guimenu]``Port``::
+Port of the remote machine providing access to the pool.
+
+[guimenu]``iSCSI Qualified Name``::
+iSCSI Qualified Name of the iSCSI target.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
Qualified name of the iSCSI target.
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 6]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+[guimenu]``IQN Initiator``::
+iSCSI Qualified Name of the initiator to connect to.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
iSCSI qualified name of the initiator to connect to.
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 7]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+[guimenu]``Username``::
+Username to use to connect to the remote storage
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
Username to use to connect to remote storage.
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 8]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+[guimenu]``Passphrase``::
+Password to use to connect to the remote storage.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
Password to use to connect to remote storage.
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 9]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+Name of the storage pool source.
+
+[guimenu]``Directory``::
+Path to the directory of the pool.
+
+[guimenu]``Subdirectory``::
+Absolute path relative to the Gluster volume to use.
+
+[guimenu]``Adapter type``::
+Select the controller type, either``fc_host`` or ``scsi_host``.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
The controller type, either``fc_host`` or ``scsi_host``.
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 10]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+[guimenu]``Adapter name``::
+SCSI adapter name for ``scsi_host`` controller.
+
+[guimenu]``Adapter parent PCI address``::
+PCI address of the SCSI host in the ``0000:00:00.0`` format.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
PCI address of the SCSI host in ``0000:00:00.0`` format.
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 11]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+Those can be listed using [systemitem]``lsscsi -v``.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
List options with [command]``lsscsi -v``.
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 12]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+[guimenu]``Permission mode``::
+Octal representation of the permissions to set on the path folder or file.
+
+[guimenu]``SELinux label``::
+SELinux label to set on the path folder or file.
+
+=== Editing a pool
+
+To edit the properties of storage pool, click the [guimenu]``Edit pool`` button in the pool's row.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
To edit the properties of a storage pool, locate the pool in the list and click [guimenu]``Edit pool``.
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 13]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+A dialog with the same fields than when creating a pool will be opened.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 14]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+=== Deleting a pool
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
=== Deleting a Pool
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 15]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+To delete a storage pool, click the [guimenu]``Delete`` button in the pool's row.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
To delete a storage pool, locate the pool in the list and click [guimenu]``Delete``.
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 16]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+By default the action only removes the storage pool definition but keeps the data.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
By default, deleting a pool only removes the storage pool definition.
The pool data is kept on disk.
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 17]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+To really clear the data, click the checkbox in the opened modal dialog before click the [guimenu]``Delete`` confirmation button.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
To delete the pool data as well the storage pool definition, check the [guimenu]``something`` box before you click [guimenu]``Delete``.
```

Need to check the name of the box.

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 18]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+Note that actually deleting the volumes and pool is depending on the pool type.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
Some pool types will not allow you to delete the volumes or the pool.
```

**Comment by Loquacity** [comment]
@cbosdo if you're happy with these suggestions, can you please accept them? Then we can get this merged.

File: modules/client-configuration/pages/virt-xenkvm.adoc [hunk 19]
```diff
@@ -35,11 +35,15 @@ For more on client registration, see xref:client-configuration:registration-over
 For Salt clients, you will need to enable the [systemitem]``Virtualization Host`` entitlement.
 This allows you to see VM changes instantly.
 To do this, in the {productname} {webui}, navigate to the [guimenu]``System Details`` page for the host, and click on the [guimenu]``Properties`` tab.
+Alternatively the [systemitem]``Virtualization Host`` entitlement can be added at the registration key level.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/client-configuration/pages/virt-xenkvm.adoc]: ACK

File: modules/client-configuration/pages/virt-xenkvm.adoc [hunk 20]
```diff
@@ -35,11 +35,15 @@ For more on client registration, see xref:client-configuration:registration-over
...
 In the [guimenu]``Add-On System Types`` section, check [guimenu]``Virtualization Host``, and click btn:[Update Properties] to save the changes.
-You will need to schedule a hardware refresh to activate the change.
-Navigate to menu:System Details[Hardware], and click btn:[Schedule Hardware Refresh].
+You will need to restart the Salt minion service to activate the change.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/client-configuration/pages/virt-xenkvm.adoc]: ACK

File: modules/client-configuration/pages/virt-xenkvm.adoc [hunk 21]
```diff
@@ -35,11 +35,15 @@ For more on client registration, see xref:client-configuration:registration-over
...
 
-By default, VM hosts use the [systemitem]``rhnsd`` service to check for scheduled actions every four hours, in order to load balance in environments where there are a lot of clients.
+----
+systemctl restart salt-minion
+----
+
+By default for tradidional clients, VM hosts use the [systemitem]``rhnsd`` service to check for scheduled actions every four hours, in order to load balance in environments where there are a lot of clients.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/client-configuration/pages/virt-xenkvm.adoc]: ACK

File: modules/client-configuration/pages/virt-xenkvm.adoc [hunk 22]
```diff
@@ -237,10 +239,10 @@ If a guest installation starts while another one is still running, the running i
 == Manage VM Guests
 
 
-You can use the {productname} {webui} to manage your VM Guests, including actions like shutting downa nd restarting, and adjusting CPU and memory allocations.
+You can use the {productname} {webui} to manage your VM Guests, including actions like shutting down, restarting, and adjusting CPU and memory allocations.
 
 To do this, you will need your Xen or KVM VM host registered to the {productname} Server, and have the [daemon]``libvirtd`` service running on the host.
-You will also need the [package]``mgr-cfg-actions`` package installed on your {productname} Server.
+For tradidional clients, you will also need the [package]``mgr-cfg-actions`` package installed on your {productname} Server.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/client-configuration/pages/virt-xenkvm.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 23]
```diff
@@ -3,13 +3,15 @@
 
 This tab allows you to create new virtual guests, apply images on a
 traditionally managed host system, or change the status of virtual guests.
+It also allows to list an manage the storage pools that are used for the virtual machines.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 10 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 14 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 15 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 16 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 17 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: Shouldn't it be `as well as the storage pool definition` ?

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 18 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

**Comment by Loquacity** [comment]
> @cbosdo if you're happy with these suggestions, can you please accept them? Then we can get this merged.

@

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 11 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 12 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 13 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 17 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: The checkbox label is `Delete the pool, including the contained volumes. Cannot be undone`. Not sure if you want to have the complete label in the doc.

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 24]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
 image::system_details_traditional_virtualization_deployment.png[scaledwidth=80%]
 
 When the deploment scheduled it is listed as an action on the menu:Main Menu[Schedule > Pending Actions].
+
+== Storage [Salt]
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: What do you mean? This describes the `Storage` sub-tab in the `Virtualization` tab.

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 25]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+The [guimenu]``Storage`` tab presents a tree list of the virtual storage pools and volumes defined on the virtual host.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 26]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+The first level of the tree is the list of storage pools and all items contained in them are volumes.
+By default the pools need to be expanded to show the volumes.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 27]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+image::modules/reference/assets/images/system_details_salt_virtual_pools.png[scaledwidth=80%]
+
+For each pool the following information is provided:
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 28]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+[guimenu]``Status``::
+This field indicates whether the pool is running or stopped.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 29]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+[guimenu]``Autostart``::
+This field indicates whether the pool is started automatically when the virtual host boots
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 30]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+[guimenu]``Persistent``::
+This field indicates whether the pool will be kept or not after being stopped.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 31]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+[guimenu]``Location``::
+This field indicates the target path of the storage pool.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 32]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+Note that some pool types don't have any associated path.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 33]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+[guimenu]``Usage``::
+This field shows the disk usage of the pool or [guimenu]``Unknown`` if the pool is not running.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 34]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+For each volume the following information is provided:
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 35]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+* The names of the virtual machines using the volume.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 36]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+This list may not be provided depending on the pool type.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 37]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+
+* The disk usage of the volume.
+
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: OK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 38]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+=== Refreshing a pool
+
+[systemitem]``Libvirt`` doesn't update the pool usage and contents in real time.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 39]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+In order to update the usage or to see a volume that has been created outside of {productname}, the pool needs to be refreshed.
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 40]
```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+Click the [guimenu]``Refresh`` button to schedule a refresh of the pool.
+
+=== Creating a pool
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 1 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: OK, how is that supposed to appear? prefixed with `[procedure]`?

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 2 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: It's a field group in the dialog. I leave it up to you how to group those in the doc, if we want them grouped there at all

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 3 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 4 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 5 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 6 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 7 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 8 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 9 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

**Comment by cbosdo** [comment]
How should I proceed for the suggestions? Accept them in Github or do you have them already stashed in a local copy?

**Comment by Loquacity** [comment]
> How should I proceed for the suggestions? Accept them in Github or do you have them already stashed in a local copy?

Just accept them here in github.

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 24 above)
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: I was referring to the markup. Why do you have Salt in square brackets?

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 41]
```diff
@@ -119,3 +121,177 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+The disk usage of the pool.
+Shows ``Unknown`` if the pool is not running.
+
+Each volume shows:
+
+* The name of the virtual machines using the volume.
+Some pool types will not provide this list.
+
+* The disk usage of the volume.
+
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion



```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 42]
```diff
@@ -119,3 +121,179 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
...
+The earliest time the pool creation action should be scheduled.
+
+[guimenu]``Add to``::
+Add the pool creation action to a new or already created action chain.
+
+All the remaining fields are depending on the selected field type.
+These fields are grouped in two panels:
+
+* the [guimenu]``Source`` panel contains the data about the device holding the pool.
+* the [guimenu]``Target`` panel contains the data about where to find the pool on the virtual host.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
.Procedure: Creating a Pool

. Click btn:[Create Pool] 
. This opens a new page with a form to define the pool.
. In the [guimenu]``name`` field, type a name for the new pool.
. In the [guimenu]``type`` field, select the type of the pool.
The list of available types depends on the virtual host setup.
// TODO Add a note on the supported types pointing to SUSE Documentation
. Check the [guimenu]``Start during virtual host boot`` field, to start the pool automatically when the virtual host boots.
. OPTIONAL: In the [guimenu]``Earliest`` field, you can set the earliest time the pool creation action should be scheduled.
. OPTIONAL: In the [guimenu]``Add to`` field, you can select a new or existing action chain to add the pool creation action to.
. The [guimenu]``Source`` section contains data about the device holding the pool.
. The [guimenu]``Target`` section contains data about where to find the pool on the virtual host.
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: I think leave it how it is for now, I'll need to go over this section once it's in to do the final edit.

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: I've edited, please confirm.

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 42 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: ACK

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 24 above)
[the remark this answers is not in the document]
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: to mimic the other sections and indicate this is a Salt-only feature. Feel free to present it how you like.

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc (hunk 1 above)
Comment (cbosdo) [reply modules/reference/pages/systems/system-details/sd-virtualization.adoc]: it's good for me

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 43]
```diff
@@ -119,3 +121,164 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
 image::system_details_traditional_virtualization_deployment.png[scaledwidth=80%]
 
 When the deploment scheduled it is listed as an action on the menu:Main Menu[Schedule > Pending Actions].
+
+== Storage [Salt]
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc]:
```suggestion
== Storage for Salt Clients
```

**Pull request merged**